File: examples/generator.py
import random
import sys

random.seed(42)
while(1):   
    data = sys.stdin.readline()
    if (data != ""):
        for x in range(0, 10000):
            line = ""
            for i in range(0, 100):
                temp = '%.12f' % random.random() 
                line += str(temp) + "\n"
            print(line,end='', flush=True)
    

File: examples/generator.r
set.seed(42)
f <- file("stdin")
open(f, blocking=FALSE)
options(scipen = 999)
while(1) {
    
    if (length(line <- readLines(f,ok=TRUE,n=1, skipNul=TRUE)) > 0) {
    for (i in 1:1000000) {
        temp = paste(format(round(runif(1), digits=12), nsmall = 12),"\n",
            format(round(runif(1), digits=12), nsmall = 12),"\n",
            format(round(runif(1), digits=12), nsmall = 12),"\n",
            format(round(runif(1), digits=12), nsmall = 12),"\n",
            sep="")
        cat(temp)
    }
    }
    
}